### DISCUSSION TITLE
Qual modelo de currículo vocês usam?!

### DISCUSSION THREAD

**Discussion by antoniotex** [frontendbr/forum#1297] [Discussão]
Pessoal, estou busca de vaga e surgiu essa dúvida.

Qual modelo de currículo usar ?
Vocês, que analisam currículo em sua empresa, qual modelo preferem ?

Eu sempre enviei o modelo convencional, sem design, etc.
Será que o modelo convencional é mau visto pelas empresas ?

**Comment by ViniciusGularte** [comment]
Eu sempre usei um modelo bem simples que fiz no Photoshop,  acredito que o foco é ter suas informações claras, dando destaque as suas experiências e qualificações, ao que você agregou, e acredito que pode ser um modelo mais convencional com tanto  que tenha todas suas informações bem claras( sempre com link do git no footer :smile: ), e não acredito que modelo convencional seja mal visto, porém trabalhei numa loja e recebíamos na média 5 currículos por dia, 90% era igual, até mesmas frases de motivação , acho que temos que dar nosso toque de originalidade para se destacarmos dos demais.

Eu deixo um currículo genérico no meu [Sitezinho](https://viniciusgularte.github.io/)  mas cada empresa penso de uma forma diferente como apresentar esse currículo, um campo a mais ou a menos, e informações cruciais para mostrar a empresa que mereço a vaga, se queremos ser notados temos que fazer por onde.

**Comment by Sergioamjr** [comment]
Eu uso um modelo de cv tradicional em .pdf, quando envio pra algum recrutador por e-mail. Mas se for um desenvolvedor ou empresa da área que irá me avaliar, eu mando [esse modelo](https://github.com/Sergioamjr/awesome-cv-template) que eu fiz, mais elaborado. Peguei o modelo no Pinterest.

**Comment by felipefialho** [comment]
Faz uns anos que só uso LinkedIn como CV...

**Comment by deppbrazil** [comment]
### No meu ponto de vista ###
Acho que independente do formato todos os cv's devem seguir um mesmo propósito de ser objetivo, pois mesmo para os recrutadores hoje em dia tempo é algo precioso e ninguém merece ficar lendo folhas e folhas de cv, então tente ser o mais direto possível, colocando apenas o que de fato agregue valor, 

### Sobre estrutura ###
> Eu normalmente separo as infos em blocos
-  **Começar com as informações pessoais:** nome, email pessoal, telefone, site/portfólio, local onde mora (cidade, estado, país) 
- **Objetivo: para qual área de desenvolvimento:** (frontend, backend, mobile, devOps, etc...) 
- **Sua formação:** graduação, pós-graduação, cursos técnicos, etc...
- **Idiomas:** português nativo, inglês intermediário, etc...
- **Stacks:** React, Vue.js, HTML5, CSS3, etc...
- **Softs Skills:** suas características pessoais, ou seja sua personalidade
- **Hard Skills:** suas competências técnicas
- **Projetos que contribuiu:** link do Github de um projeto que tenha contribuído
- **Experiências anteriores:** empresa, cargo, tempo, responsabilidades 

**⏭  Ahh ter uma versão em inglês do cv também conta bastante**

### Formatos ###
Quanto ao formato ser digital, pdf, etc... varia muito pois tem empresas que irá pedir para anexar no email a versão pdf em inglês, outras não farão questão da versão em inglês, tem empresa que irá pedir o link do portfólio, outras que entram em contato direto pelo Linkedin... varia muito.... 

Eu recentemente criei um repo no Github com meu cv, então quando preciso enviar para alguém já tenho os formatos que preciso sempre atualizados e versionados, e quando preciso atualizar uma informação fica mais fácil... 😃 
[Meu cv](https://github.com/deppbrazil/cv)

### Fica minha dica 😄 ###

**Comment by merciakeila** [comment]
> ### No meu ponto de vista
> Acho que independente do formato todos os cv's devem seguir um mesmo propósito de ser objetivo, pois mesmo para os recrutadores hoje em dia tempo é algo precioso e ninguém merece ficar lendo folhas e folhas de cv, então tente ser o mais direto possível, colocando apenas o que de fato agregue valor,
> 
> ### Sobre estrutura
> > Eu normalmente separo as infos em blocos
> 
> * **Começar com as informações pessoais:** nome, email pessoal, telefone, site/portfólio, local onde mora (cidade, estado, país)
> * **Objetivo: para qual área de desenvolvimento:** (frontend, backend, mobile, devOps, etc...)
> * **Sua formação:** graduação, pós-graduação, cursos técnicos, etc...
> * **Idiomas:** português nativo, inglês intermediário, etc...
> * **Stacks:** React, Vue.js, HTML5, CSS3, etc...
> * **Softs Skills:** suas características pessoais, ou seja sua personalidade
> * **Hard Skills:** suas competências técnicas
> * **Projetos que contribuiu:** link do Github de um projeto que tenha contribuído
> * **Experiências anteriores:** empresa, cargo, tempo, responsabilidades
> 
> **⏭ Ahh ter uma versão em inglês do cv também conta bastante**
> 
> ### Formatos
> Quanto ao formato ser digital, pdf, etc... varia muito pois tem empresas que irá pedir para anexar no email a versão pdf em inglês, outras não farão questão da versão em inglês, tem empresa que irá pedir o link do portfólio, outras que entram em contato direto pelo Linkedin... varia muito....
> 
> Eu recentemente criei um repo no Github com meu cv, então quando preciso enviar para alguém já tenho os formatos que preciso sempre atualizados e versionados, e quando preciso atualizar uma informação fica mais fácil... 😃
> [Meu cv](https://github.com/deppbrazil/cv)
> 
> ### Fica minha dica 😄

Seu currículo é incrível! Estarei usando como inspiração. Fiquei curiosa em relação ao Codersranking. Você já conseguiu algum job por lá para homeoffice?

**Comment by deppbrazil** [comment]
@merciakeila sorry pela demora... :| 

Fico muito feliz que meu modelo tenha servido de inspiração... :) 

Sobre o Codersranking não cheguei a conseguir jobs por lá, sempre aparece sugestões, mas meu inglês ainda não é fluente e fico meio inseguro, então uso as métricas geradas para portfólio mesmo e acho legal colocar isso no currículo, é uma forma também de ser avaliado por repositórios em questão de organização, código e ranking.

**Comment by jair-anderson-souza** [comment]
@Sergioamjr teu link tá quebrado

**Comment by Sergioamjr** [comment]
@jass2125 atualizei, é esse alias https://github.com/Sergioamjr/awesome-cv-template
